GoogleForCreators/web-stories-wp
# Disable the button for saving a page template for a blank page

## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions
1. Add a blank page
2. Open the Page templates tab in the library
3. Verify the button for saving the page as template is disabled
4. Now add any element to the page
5. Verify the button appears and works as expected.


### QA
See the steps above.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #7498
